packages/gnosis/src/index.ts

```diff
@@ -37,7 +38,7 @@ function gnosis(options?: GnosisOptions): WalletInit {
               SafeAppsSDK.default || SafeAppsSDK
 
             const opts = {
-              whitelistedDomains
```

Comment:
Maybe it's better to do it this way? So that the interface doesn't change
```
const opts = {
  allowedDomains: whitelistedDomains
}
```

Reply:
Yes, we would like to keep the interface consistent and compatible through versions